packages/pyml_bindgen/pyml_bindgen.0.3.1/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "Generate pyml bindings from OCaml value specifications"
+maintainer: "Ryan M. Moore"
+authors: "Ryan M. Moore"
+license: ["MIT" "Apache-2.0"]
+homepage: "https://github.com/mooreryan/ocaml_python_bindgen"
+doc: "https://mooreryan.github.io/ocaml_python_bindgen/"
+bug-reports: "https://github.com/mooreryan/ocaml_python_bindgen/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "angstrom" {>= "0.15.0"}
+  "base" {>= "v0.12"}
+  "cmdliner" {>= "1.1.0"}
+  "ppx_let" {>= "v0.12"}
+  "ppx_sexp_conv" {>= "v0.12"}
+  "ppx_string" {>= "v0.12"}
+  "re" {>= "1.5"}
```

Comment:
```suggestion
  "re" {>= "1.7.2"}
```
Solves `Unbound module Re.Perl`. This is the version that introduced the `Re.` modules in place of `Re_`